examples/deployment/kubernetes/trillian-map-deployment.yaml

```diff
@@ -1,44 +1,80 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  annotations:
-    kompose.cmd: kompose convert
-    kompose.version: 1.3.0 (HEAD)
-  creationTimestamp: null
   labels:
     io.kompose.service: trillian-map
-  name: trillian-map
+  name: trillian-mapserver-deployment
 spec:
-  replicas: 1
+  replicas: 4
   strategy: {}
   template:
     metadata:
-      creationTimestamp: null
       labels:
         io.kompose.service: trillian-map
     spec:
+      volumes:
+      - name: google-cloud-key
+        secret:
+          secretName: trillian-key
+      restartPolicy: Always
       containers:
-      - command:
-        - /go/bin/trillian_map_server
-        - --mysql_uri=test:zaphod@tcp(db:3306)/test
-        - --rpc_endpoint=0.0.0.0:8090
-        - --http_endpoint=0.0.0.0:8091
-        - --alsologtostderr
-        image: us.gcr.io/project-name/map_server:latest
+      - name: trillian-map
+        args: [
+        "$(STORAGE_FLAG)",
+        "--storage_system=$(STORAGE_SYSTEM)",
+        "--etcd_servers=trillian-etcd-cluster-client:2379",
+        "--quota_system=etcd",
+        "--rpc_endpoint=0.0.0.0:8090",
+        "--http_endpoint=0.0.0.0:8091",
+        "--tracing",
+        "--alsologtostderr"
+        ]
+        envFrom:
+          - configMapRef:
+              name: deploy-config
+        image: gcr.io/${PROJECT_NAME}/map_server:latest
         imagePullPolicy: Always
+        resources:
+          limits:
+            cpu: "1.0"
         livenessProbe:
-          exec:
-            command:
-            - curl
-            - -f
-            - http://localhost:8091/healthz
+          httpGet:
+            path: /healthz
+            port: 8091
           failureThreshold: 3
           periodSeconds: 30
-          timeoutSeconds: 30
-        name: trillian-map
+          timeoutSeconds: 5
         ports:
         - containerPort: 8090
+          name: rpc
         - containerPort: 8091
-        resources: {}
-      restartPolicy: Always
-status: {}
```

Comment:
What does `status:` do?  (I notice it's still in the equivalent log-deployment.yaml file)

Reply:
"someone" - I plead guilty :-) 

`docker-compose` can't support side-cars, so we ditched it.